### DISCUSSION TITLE
Are there any news on the pool-farming protocol?

### DISCUSSION THREAD

**Discussion by SackofFluor** [Chia-Network/chia-blockchain#6946] [Support]
Are there any news on the pool-farming protocol?

Or did the dev team abandoned that, now seeing that they coin collapsed in value and they are in hpool anyways?

**Comment by gromran** [comment]
This is lobbying by the whales, if they don't want that, it won't happen either. It's the same as PoS at ETH

**Reply by loppefaaret** [reply]
come on now, how far behind is PoS for ETH ? and how far behind is pooling for chia so far ?

**Comment by lvcivs** [comment]
Work is ongoing, you can check out progress on github if you know where to look.

**Reply by iaimit** [reply]
请问这个进度怎么看？

**Reply by vazmuten** [reply]
Would you please enlighten us where to clearly see the progress bar?

**Reply by loppefaaret** [reply]
you can currently follow any of the "pool" branches under our https://github.com/Chia-Network/chia-blockchain/ repo

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
